mappings/net/minecraft/client/render/BaseFrustum.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_84 net/minecraft/client/render/BaseFrustum
+	FIELD field_285 frustum [[F
+	FIELD field_286 projectionMatrix [F
+	FIELD field_287 modelViewMatrix [F
```

Comment:
couldn’t this just be viewMatrix? Or is this the model transform? In which case something else